packages/application-generic/src/usecases/create-notification-jobs/create-notification-jobs.usecase.ts

```diff
@@ -34,7 +34,9 @@ export class CreateNotificationJobs {
   constructor(
     private digestFilterSteps: DigestFilterSteps,
     private notificationRepository: NotificationRepository,
+    @Inject(forwardRef(() => CalculateDelayService))
     private calculateDelayService: CalculateDelayService,
+    @Inject(forwardRef(() => EventsPerformanceService))
```

Comment:
Forced to implement this. Need to investigate what's the circular dependency causing this.